beta/src/components/Layout/Nav/MobileNav.tsx

```diff
@@ -4,9 +4,9 @@
 
 import * as React from 'react';
 import cn from 'classnames';
-import {RouteItem} from 'components/Layout/useRouteMeta';
-import {useRouter} from 'next/router';
-import {SidebarRouteTree} from '../Sidebar';
+import { RouteItem } from 'components/Layout/useRouteMeta';
```

Comment:
These changes make it harder to review what's actually changed in this PR. You can run `yarn prettier` in the `beta` folder to get rid of these.

Reply:
ok i ran the command